sympy/printing/pycode.py

```diff
@@ -822,6 +822,9 @@ def _print_Mod(self, expr):
         return "%s(%s)" % (self._module_format('numpy.mod'), ', '.join(
             map(lambda arg: self._print(arg), expr.args)))
 
+    def _print_heaviside(self, expr):
+        return "%s(%s,%s)" % (self._module_format('numpy.sinc'), self._print(expr.args[0], self._print(0)))
```

Comment:
`sinc`? Also, I think the method name should read `_print_Heaviside`? Is 0 used for Heaviside(0) in any of the other backends? I think it's fine using zero, we could add a printer option in the future if there are requests for that. I would also say that we should add a test to the test suite of lambdify, where results from e.g. -1, 0 & 1 are tested.

Reply:
I had troubles comprehending heaviside(0) with printers with some attempts before so I’m fine with making it agnostic with the actual sympy value.
I also think that heaviside(0) should better be replaced with a named symbol than wrapping around the function itself.

Reply:
@bjodah, you're absolutely right. I made the modifications in-place in my Sympy installation first to see if it worked, then when I made this pull request I apparently got lazy. I'll fix that. As far as a test case, I'm a little unfamiliar with exactly how to make that modification, so guidance would be appreciated.

@sylee957, I have now included a variable called `heaviside0` inside the printing function. We could move that to a larger scope if necessary.

Reply:
@Ickaser you can add a test analogous to https://github.com/sympy/sympy/blob/4f95487032a53342bdef5b76b566f59b381fb6e3/sympy/utilities/tests/test_lambdify.py#L999

and regarding a symbol for Heaviside(0) I think @sylee987 was referring to `class Heaviside0`, analogous to e.g. https://github.com/sympy/sympy/blob/c60bf732d40a5cd2060ce66deeb79adecc13408d/sympy/core/numbers.py#L3767
that way, a printer may implement `_print_Heaviside0`.

Reply:
I don't think I understand the purpose of the Heaviside0 symbol class quite well enough to fully implement that, but I have made a good-faith attempt to do so. I tried to explain in the description of the class that the value depends on convention, but currently returns 0 if evaluated. I wondered if it should return the Sympy Heaviside(0) object, but didn't really know how to implement that.

Reply:
I'll let @sylee957 comment on a potential class `Heaviside0` (I'm fine with leaving it out of this pull request, and deferring to a later time, in the interest of getting `lambdify` support of Heaviside).

Reply:
I think that it can make the PR potentially grow bigger and make many test cases fail for now.
I’d leave Heaviside0 for other PR

Reply:
Alright, I have reverted that commit. Is there anything else that needs to be changed for this PR?